### ISSUE TITLE
Improve article 2204.11824

### ISSUE THREAD

**Issue by dginev** [dginev/ar5iv#174]
**Exact location of issue**

Titles! Many, many titles.

**Problem details**

Screenshot of the HTML title page:

<details>
<img src="https://user-images.githubusercontent.com/348975/182657048-8e2d7e62-256a-4cbc-bbef-f00f3059bb2a.png" height="800">
</details>

From the main article `ms.tex` source:
```tex
\title{Semi-Parametric Neural Image Synthesis}
\title{Semi-Parametric Diffusion Models}
\title{Retrieval-Augmented Image Synthesis in Latent Space}
\title{Retrieval-Augmented Latent Diffusion Models}
\title{Retrieval-Augmented Diffusion Models in Latent Space}
\title{Retrieval-Augmented Diffusion Modeling in Latent Space}
\title{Retrieval-Augmented Diffusion Modeling}
\title{Retrieval-Augmented Diffusion Models}
\title{Retrieval-Augmented Neural Diffusion}
\title{Retrieval-Augmented Diffusion Models}
```

LaTeXML should only use the last defined title.

**Comment by dginev** [comment]
A related article with this problem is [2111.15588](https://ar5iv.labs.arxiv.org/html/2111.15588) (transferring from [engrafo#1316](https://github.com/arxiv-vanity/engrafo/issues/1316) ).

Again a case of two title macros:
```tex
\title{A template for the \emph{arxiv} style}
%...
\title{SimpleTRON: Simple Transformer with O(N) Complexity}
```

Should get resolved with [latexml#1937](https://github.com/brucemiller/LaTeXML/pull/1937)

**Comment by dginev** [comment]
Fixed in 2023.
